src/main/java/no/api/freemarker/java8/time/DateTimeTools.java

```diff
@@ -113,4 +114,12 @@ public static ZoneId zoneIdLookup(List list, int index) throws TemplateModelExce
         return zoneId;
     }
 
+    private static Locale getLocale() {
+        if (Environment.getCurrentEnvironment() != null) {
+            return Environment.getCurrentEnvironment().getConfiguration().getLocale();
```

Comment:
Not completely familiar with Freemarker... Is there any way `getConfiguration()` could return `null`, thus causing a `NullPointerException`?